lib/axiom/fs/base/execute_context.js

```diff
@@ -213,6 +205,27 @@ ExecuteContext.prototype.setStdio = function(stdio) {
 };
 
 /**
+ * @return {!Stdio}
+ */
+ExecuteContext.prototype.getStdio = function() {
```

Comment:
This is needed because:
```
cx.setStdio(stdio1);
...  
cx.stdin = stdin2;
...
cx.stdio.stdin.addListener(); // attaches to stdio1.stdin
cx.stdin.addListener(); // attaches to stdin2
```
I would get rid of `ExecuteContext.getStdio()` as well, but some client code still wants to apply group actions to the whole virtual `ExecuteContext.stdio` at once.